web-common/src/features/dashboards/proto-state/sparse-proto.spec.ts

```diff
@@ -103,7 +103,10 @@ describe("sparse proto", () => {
             AD_BIDS_TIME_RANGE_SUMMARY,
           ),
         );
-        const defaultProto = getProtoFromDashboardState(dashboard);
+        const defaultProto = getProtoFromDashboardState(
+          dashboard,
+          AD_BIDS_EXPLORE_INIT,
```

Comment:
Nit: this variable name should include `SPEC`

Reply:
This is a greater refactor of the names in test. Lets not do it in this PR.